staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -902,8 +902,17 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 			requestScopes[v.Name] = &reqScope
 		}
 
-		// override scaleSpec subresource values
+		// override scale subresource values
 		// shallow copy
+		scaleColumns, err := getScaleColumnsForVersion(crd, v.Name)
+		if err != nil {
+			utilruntime.HandleError(err)
+			return nil, fmt.Errorf("the server could not properly serve the CR scale subresource columns")
+		}
+		scaleTable, err := tableconvertor.New(scaleColumns)
+		if err != nil {
+			klog.V(2).Infof("The CRD for %v has an invalid printer specification, falling back to default printing: %v", kind, err)
```

Comment:
structured logging?

How can this case happen?

Reply:
This case will never happen because the scale columns are always defined in code and cannot be overridden by the user. 

Will change this to ignore the error here.